### PR TITLE
Tests improved

### PR DESCRIPTION
**Pull request by marclll**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           

Coverage increased per file
===========================
+ src/main/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoImpl.java  18
+ src/main/java/org/pesmypetcare/webservice/entity/petmanager/PetEntity.java  8
+ src/main/java/org/pesmypetcare/webservice/entity/appmanager/EventEntity.java  20
         

Coverage decreased per file
===========================
- src/main/java/org/pesmypetcare/webservice/service/appmanager/GoogleCalendarServiceImpl.java  -5
         

Complexity increasing per file
==============================
- src/main/java/org/pesmypetcare/webservice/service/appmanager/GoogleCalendarServiceImpl.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496&bid=18078480)

File: src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java [hunk 1]
```diff
@@ -57,10 +61,40 @@
...
 
+    @BeforeEach
+    public void setUp() {
+        petEntity = new PetEntity(GenderType.Female, "Huskie", "1996-01-08T12:20:30",
+            "One limb lost", "Constant attention", 85.0, null,
+            null);
+        petList = new ArrayList<>();
+        Map<String, Object> auxMap = new HashMap<>();
+        auxMap.put("name", PET_NAME);
+        auxMap.put("body", petEntity);
```
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java:78]: Codacy Issue found: [The String "body" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)

File: src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java [hunk 2]
```diff
@@ -57,10 +61,40 @@
...
+        petList.add(auxMap);
+        petList.add(auxMap);
+        petList.add(auxMap);
+        snapshotList = new ArrayList<>();
+        snapshotList.add(documentSnapshot);
+        snapshotList.add(documentSnapshot);
+        snapshotList.add(documentSnapshot);
+        mealMap = new HashMap<>();
+        mealMap.put("mealName", "Asparagus");
+        mealMap.put("kcal", 84.0);
```
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java:88]: Codacy Issue found: ['84.0' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)

File: src/test/java/org/pesmypetcare/webservice/dao/appmanager/GoogleCalendarDaoTest.java [hunk 3]
```diff
@@ -70,9 +71,16 @@ public static void setUp() {
         eventId = "eventId";
         calendar = new Calendar();
         calendar.setId("calendarId");
-        event = new Event();
+        EventEntity auxEventEntity = new EventEntity("id1", "Summary", "Description", "Spain", "2",
+            10, 5, "1996-01-08T10:20:30", "1996-01-08T12" +
```
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/dao/appmanager/GoogleCalendarDaoTest.java:75]: Codacy Issue found: ['+' should be on a new line.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)

File: src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java [hunk 4]
```diff
@@ -57,10 +61,40 @@
...
+    @Mock
+    private DocumentSnapshot documentSnapshot;
 
     @InjectMocks
     private PetDao petDao = new PetDaoImpl();
 
+    @BeforeEach
+    public void setUp() {
+        petEntity = new PetEntity(GenderType.Female, "Huskie", "1996-01-08T12:20:30",
+            "One limb lost", "Constant attention", 85.0, null,
```
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java:73]: Codacy Issue found: ['85.0' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)

File: src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java [hunk 5]
```diff
@@ -57,10 +61,40 @@
     private WriteBatch batch;
     @Mock
     private StorageDao storageDao;
+    @Mock
+    private DocumentSnapshot documentSnapshot;
 
     @InjectMocks
     private PetDao petDao = new PetDaoImpl();
 
+    @BeforeEach
```
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java:70]: Codacy Issue found: [NCSS for this method is 23 (max allowed is 15).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)

File: src/main/java/org/pesmypetcare/webservice/service/appmanager/GoogleCalendarServiceImpl.java [hunk 6]
```diff
@@ -29,6 +29,9 @@
     @Override
     public void createSecondaryCalendar(String accessToken, String owner, String petName)
         throws CalendarAccessException, DatabaseAccessException, DocumentException {
+        if (petDao.getSimpleField(owner, petName, CALENDAR_ID) != null) {
+            throw new CalendarAccessException("409", "This pet already has a secondary calendar");
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/service/appmanager/GoogleCalendarServiceImpl.java:33]: Codacy Issue found: [The String "409" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)

File: src/test/java/org/pesmypetcare/webservice/dao/appmanager/GoogleCalendarDaoTest.java (hunk 3 above)
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/dao/appmanager/GoogleCalendarDaoTest.java:75]: Codacy Issue found: ['5' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)

File: src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java (hunk 5 above)
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java:70]: Codacy Issue found: [Method length is 26 lines (max allowed is 15).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Coverage increased per file
===========================
+ src/main/java/org/pesmypetcare/webservice/entity/petmanager/PetEntity.java  8
+ src/main/java/org/pesmypetcare/webservice/entity/appmanager/EventEntity.java  20
+ src/main/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoImpl.java  18
         

Coverage decreased per file
===========================
- src/main/java/org/pesmypetcare/webservice/service/appmanager/GoogleCalendarServiceImpl.java  -5
         

Complexity increasing per file
==============================
- src/main/java/org/pesmypetcare/webservice/service/appmanager/GoogleCalendarServiceImpl.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496&bid=18078480)

File: src/test/java/org/pesmypetcare/webservice/dao/appmanager/GoogleCalendarDaoTest.java [hunk 7]
```diff
@@ -70,9 +71,16 @@ public static void setUp() {
         eventId = "eventId";
         calendar = new Calendar();
         calendar.setId("calendarId");
-        event = new Event();
+        EventEntity auxEventEntity = new EventEntity("id1", "Summary", "Description",
+            "Spain", "2", 10, 5, "1996-01-08T10:20:30",
```
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/dao/appmanager/GoogleCalendarDaoTest.java:75]: Codacy Issue found: ['5' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ src/main/java/org/pesmypetcare/webservice/entity/petmanager/PetEntity.java  8
+ src/main/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoImpl.java  18
+ src/main/java/org/pesmypetcare/webservice/entity/appmanager/EventEntity.java  20
         

Coverage decreased per file
===========================
- src/main/java/org/pesmypetcare/webservice/service/appmanager/GoogleCalendarServiceImpl.java  -5
         

Complexity increasing per file
==============================
- src/main/java/org/pesmypetcare/webservice/service/appmanager/GoogleCalendarServiceImpl.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496&bid=18078480)

**Review by AlvarTB — APPROVED** [review]

**Review by santidrj — APPROVED** [review]
:+1:

**Pull request merged**